src/main/java/no/nav/tag/sykefravarsstatistikk/api/TestController.java

```diff
@@ -0,0 +1,23 @@
+package no.nav.tag.sykefravarsstatistikk.api;
+
+import lombok.extern.slf4j.Slf4j;
+import no.nav.security.oidc.api.Unprotected;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@Unprotected
+public class TestController {
+
+    private Logger auditLogger = LoggerFactory.getLogger("auditLogger");
+
+    @GetMapping(value = "/test")
```

Comment:
Er litt usikker i utgangspunktet med tanken om å beholde et _test_ endepunkt i applikasjonen, men hvis vi gjør det, burde det være et mer selvforklarende navn som `/internal/sjekkauditlogg` kanskje

Reply:
Dette var egentlig bare for test, jeg glemte å slette den :sweat_smile: